hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: minor
+
+This release enables deprecation warnings even when the
+:obj:`~hypothesis.settings.verbosity` setting is ``quiet``,
+in preparation for Hypothesis 5.0 (:issue:`2218`).
```

Comment:
I wonder if this is a breaking change and whether we should start emitting a deprecation warning saying that the use of quiet to suppress deprecation warnings is deprecated and that you should use the standard `warnings` module mechanisms to suppress it instead if you want to see the actual warning instead of this deprecated deprecation warning.